### DISCUSSION TITLE
[2021.08.24]  위장

### DISCUSSION THREAD

**Discussion by hyk339** [pyunji/Hyundai-ITnE-Team_QuadCore#31] [Algorithm]
<JAVA 알고리즘 문제>

[위장](https://programmers.co.kr/learn/courses/30/lessons/42578)

**Comment by pyunji** [comment]
[위장](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/PR/PR_42578.java)

**Comment by hyk339** [comment]
[위장 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/programmers/PR_42578.java)

**Comment by chaeyeon4u** [comment]
[위장 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/PR_42578.java)

**Comment by leechang4** [comment]
[위장](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/lch/Programmers/PR%2042578.java)
